static-build/openssl-tarantool-security-54.patch

```diff
@@ -0,0 +1,10 @@
+--- openssl.old/ssl/statem/extensions_clnt.c	2023-02-09 16:18:41.231053686 +0300
++++ openssl/ssl/statem/extensions_clnt.c	2023-02-09 17:33:09.257282077 +0300
+@@ -266,6 +266,7 @@
+         return EXT_RETURN_NOT_SENT;
+ 
+     if (!WPACKET_put_bytes_u16(pkt, TLSEXT_TYPE_session_ticket)
++            || !s->session 
+             || !WPACKET_sub_memcpy_u16(pkt, s->session->ext.tick, ticklen)) {
```

Comment:
I think that we should just skip the call to `WPACKET_sub_memcpy_u16` if `s->session == NULL`, not fail the function.

Reply:
Agreed to leave this is as is, because we shouldn't hit this code path anyway and checking `s->session` is needed solely to suppress the SVACE warning so it doesn't really matter how we fix it.

Reply:
rebased